desc/compute/_metric.py

```diff
@@ -1340,6 +1340,30 @@ def _g_sup_rt(params, transforms, profiles, data, **kwargs):
     return data
 
 
+@register_compute_fun(
+    name="g^pa",
```

Comment:
This will cause conflict with ideal_ballooning PR. I'll add some more details but I think we should try to make then as compatible as possible.